src/resource.jl

```diff
@@ -10,17 +10,17 @@ const resource_re = Regex("""
 """, "x")
 
 """
-    get_registries(server)
+    get_registries(server, flavor)
 
 Interrogate a storage server for a list of registries, match the response against the
 registries we are paying attention to, return dict mapping from registry UUID to its
 latest treehash.
 """
-function get_registries(server::AbstractString)
+function get_registries(server::AbstractString, flavor::AbstractString)
     regs = Dict{String,String}()
-    response = HTTP.get("$server/registries", status_exception = false)
+    response = HTTP.get("$server/registries.$(flavor)", status_exception = false)
```

Comment:
How does PkgServer know if the storage server have flavors? It would be great to still support unflavored storage servers at least.

Reply:
Okay, I have added a "flavorless" mode, which is autodetected if all of its upstream storage servers don't respond to these resources.